### DISCUSSION TITLE
DevicePath - before like USB, now COMn

### DISCUSSION THREAD

**Discussion by FlemmingOstergaard** [IntergatedCircuits/HidSharp#49] [Q&A]
Hi
Somewhere in an update, the DevicePath has changed from something like "usb&hid....." to COMn.

I need to identify the actual USB device, sitting on the COM port.

Anyone know a solution to this?

**Comment by benedekkupper** [comment]
Can you give us some actual information, like what OS you are talking about? Update of the OS / device driver / .NET Framework / HidSharp library?

The most likely scenario is that your USB device's VID:PID was assigned a specific default driver by Microsoft, and it no longer relies on the device descriptors anymore. Try to change the assigned driver to generic HID device in Device Manager.
